### ISSUE TITLE
Derived Fields: Doesn't show values in Cross-Cycle comparison

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#3672]
**Describe the bug**

Values for Derived Columns do not display in Cross-Cycle comparisons
![image](https://user-images.githubusercontent.com/6314950/206330331-478e6d60-0e06-4368-ad33-ca63a4dd8010.png)

**Expected Behavior**

Expected to see values for Derived Columns in Cross-Cycle comparisons

**Actual Behavior**

See above

**Steps to Reproduce**
* Make a derived field that has values
* Make sure it is displaying in the Inventory list view
* Click on Cross-Cycles
* I would expect that the column for the derived fields will not have data

**Instance Information**

Instance: staging
SHA: bb2fdd425

**Comment by anchapin** [comment]
@haneslinger and @axelstudios will this issue be fixed after #3438 is done?

**Comment by haneslinger** [comment]
Yes, but https://github.com/SEED-platform/seed/issues/3438 is a massive lift and is an indefinite hold.
